encryption_works.md

```diff
@@ -98,25 +98,41 @@ Tails, which is a live DVD and live USB GNU/Linux distribution that I will discu
+Despite NSA surveillance, it’s still possible to browse the web with a high degree of anonymity. The Tor Browser is a web browser like Firefox or Safari, but it allows you to use the Internet while concealing your IP address, which is generally an accurate representation of your approximate location. 
 
-It's important to remember that just because your connection to the Internet may be anonymous that doesn't magically make it secure. EFF has made a great [visualization of how Tor and HTTPS can work together to protect your privacy](https://www.eff.org/pages/tor-and-https).
+The software is free and open-source, meaning that it comes with a high degree of trust, and is an adapted version of the Firefox browser.
 
-Like all good cryptography software, Tor is free software, complete with an open [bug tracker](https://trac.torproject.org/projects/tor), [mailing lists](https://www.torproject.org/docs/documentation#MailingLists), and [source code](https://gitweb.torproject.org/tor.git?a=tree;hb=HEAD).
+![The Tor Browser](https://raw.githubusercontent.com/tommycollison/encryption-works/master/images/tor_FPF.png)
 
-Documentation for Tails, the live GNU/Linux distribution that forces all of the user's network traffic to go through the Tor network, has [this to say](https://tails.boum.org/doc/about/warning/index.en.html#index7h1) about global adversaries:
+The Tor network is made up of over 6,000 volunteer servers, called nodes. When you use Tor, your connection is routed through three of these nodes —in a circuit— before exiting onto the normal Internet. If someone is monitoring your traffic, they can only track traffic through the final node, which is in a different location to you. 
 
-> A global passive adversary would be a person or an entity able to monitor at the same time the traffic between all the computers in a network. By studying, for example, the timing and volume patterns of the different communications across the network, it would be statistically possible to identify Tor circuits and thus matching Tor users and destination servers.
+![To obscure your location, Tor routes your connection through three intermediaries.](/images/tor_relays.png)
 
-We still don't know whether or not NSA or GCHQ counts as a global adversary, but we do know that they monitor a large portion of the Internet. It's too early to know for sure how often these intelligence agencies can defeat the anonymity of the Tor network.
+Despite attempts by US and UK intelligence agencies to deanonymize Tor traffic, the Tor network is still largely secure. One leaked NSA slide titled "Tor Stinks" states that the de-anonymization success rate has been [negligible](http://www.theguardian.com/world/interactive/2013/oct/04/tor-stinks-nsa-presentation-document). Another [described](http://www.theguardian.com/world/interactive/2013/oct/04/tor-high-secure-internet-anonymity) Tor as "Still the King of high secure, low latency Internet Anonymity" with "no contenders."
```

Comment:
I'd prefer to keep Micah's old paragraphs here about "global adversaries" and not call Tor "still largely secure" since these are things we actually do NOT know. There was a big discussion about this section when we first wrote it (I wish we had it on github!) and we don't want to oversell the security. It's a tough needle to thread. Can combine the new and old paragraphs like this (while keeping paragraph 109 as well):

"While some leaked NSA slides indicate the surveillance agency could not de-anonymize all of Tor's traffic in 2012 [or whenever the slides were] and called Tor "the King of high secure, low latency Internet Anonymity," we still don't know whether or not NSA or GCHQ counts as a global adversary. But we do know that they monitor a large portion of the Internet, so it's hard to fully tell for sure how often these intelligence agencies can defeat the anonymity of the Tor network."